http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -410,13 +410,19 @@ object Boot extends IOApp {
         googleComputeRetryPolicy
       )
 
+      googleDataprocRetryPolicy = RetryPredicates.retryConfigWithPredicates(
```

Comment:
Not directly related to this ticket, but added a `RetryConfig` to Dataproc. See: https://github.com/broadinstitute/workbench-libs/pull/630